llvm/test/CodeGen/AMDGPU/fmaximum3.ll

```diff
@@ -3766,3 +3766,69 @@ define <2 x double> @v_no_fmaximum3_f64__multi_use(double %a, double %b, double
   %insert.1 = insertelement <2 x double> %insert.0, double %max1, i32 1
   ret <2 x double> %insert.1
 }
+
+; Checks whether the test passes; performMinMaxCombine() should not optimize vector patterns of maximum3
```

Comment:
This is achieved by v_fmaximum3_v2f16 already? This one just has a second instance

Reply:
Good point. This had been removed upstream but for some reason lived on in our downstream branch.